### ISSUE TITLE
[Bug]: love.system.GetOS crashes Arm11

### ISSUE THREAD

**Issue by wewewer123** [lovebrew/lovepotion#217]
### Software Version

3.0.0

### What Happened?

Arm11 crashed
[crash_dump_00000000.dmp](https://github.com/lovebrew/lovepotion/files/13357103/crash_dump_00000000.dmp)

### What was Expected?

returned OS name (so in this case "Horizon")

### Code to Reproduce

```Lua
love.draw()
	love.graphics.print(love.system.getOS())
end
```

### Console

Nintendo 3DS

### Firmware Version(s)

11.17.0-50E

### Custom Firmware Version

13.0.2

### Execution Method

Homebrew Menu

### Code of Conduct

- [X] I have read the Code of Conduct and agree to these terms

**Comment by TurtleP** [comment]
You forgot to write it as:
```lua
function love.draw()
	love.graphics.print(love.system.getOS())
end
```

This otherwise does not define a Lua function, hence the crash. Currently there is an issue with text rendering when there is too much text otherwise (I need to open an issue an pin it) on 3DS and I am trying to solve it, but it's quite difficult.

**Comment by wewewer123** [comment]
You're  right, I forgot the function call... but there is some strange things going on between 3.0pre5 and 3.0. As with pre5 my code worked, but (besides 3.0 never telling me what I miscoded and just crashing my 3ds (model name "RED"), every time) now on 3.0 the exact same code doesn't work (both images and text, but music/input handling does still work) and I opened this bug report as I concluded that some function didn't do what it should anymore (I presumed GetOS). So there are some weird things going on between the 2 versions.
